orchestration/templates/ingest-hca.yaml

```diff
@@ -0,0 +1,61 @@
+apiVersion: argoproj.io/v1alpha1
+kind: WorkflowTemplate
+metadata:
+  name: ingest-hca
+spec:
+  entrypoint: main
+  serviceAccountName: {{ .Values.serviceAccount.k8sName }}
+  templates:
+    - name: main
+      inputs:
+        parameters:
+          - name: source-bucket-name
```

Comment:
I think we'll need a `prefix` along with the bucket name

Reply:
Given that each bucket is supposed to fit a particular layout (where it has the metadata, data, and links directories), what would the prefix be used for?

Reply:
The prefix gives more flexibility in storing multiple "areas" within a single bucket (i.e. EBI wants to use one bucket for both dev and prod). metadata/data/links are all relative to bucket + prefix.